README.md

```diff
@@ -26,69 +26,70 @@ Install the Docker generator:
+### Node.js
+- From the root of your project, where your server.js file existts, open a terminal window
+```bash
+$ yo docker
+```
+- Answer the prompts, including your image name
+- Open the file you wish to debug and set a breakpoint
+- From VS Code, **[Command]** + **[P]** for the command palate
+- Enter: `task composeForDebug` This will compose your containers using the dockerTask.sh script
+- **[F5]** to start the node debugger within the container
+- As we use `nodemon -L --debug-brk`, you'll see the first line of server.js hit a breakpoint, so that you can debug the starting process of your app
+
+### DotNet Core
+- From the root of your project, where your server.js file existts, open a terminal window
 ```bash
-yo docker
+$ yo docker
 ```
+- Answer the prompts, including your image name
+- Open the file you wish to debug and set a breakpoint
+- **[F5]** to start the .NET Core debugger within the container
 
 ## Contributing
 See [Contributing][contributingLink] for guidelines.
 
-## Multi-Environment and Multi-Container Compose Support - Prototype
-We are working on adding docker-compose support that will support multiple containers and multiple environments.
-`dockerTask compose dev` will issue a docker-compose up with the merged docker-compose files
-`dockerTask compose staging` will remove the volume mapping and other debug settings
-
-To see and provide feedback, please take a look at: [yo docker compose prototype][yodockerprototype]
-
 ## Q&A
 - **Q: Are you building an abstraction layer over docker apis?**
   - **A:** No. These are your scripts. We're simply providing a starting point that docker developers would write themselves. These scripts are based on customers we've been working with
 - **Q: Are you taking feedback?**
   - **A:** Of course. Please open an issue at [yodockerissues] or choose to [contribute][contributingLink]
 
 ## Collecting usage data
-Generator-docker collects anonymized data on the options you selected in the tool to understand and improve the experience. You are given a choice to
-opt-in or opt-out first time you run the tool. If you opt-in and decide to opt-out later, simply delete the `~/.config/configstore/generator-docker.json` file from your machine.
+Generator-docker collects anonymized data on the options you selected in the tool to understand and improve the experience. You are given a choice to opt-in or opt-out first time you run the tool. If you opt-in and decide to opt-out later, simply delete the `~/.config/configstore/generator-docker.json` file from your machine.
+However, we really need your feedback, so please help us help you by opting in.
 
-## Changelog
-```
-v0.0.32
-=======
+## Version History
+###v0.0.32
++ Added integrated debugging using **[F5]** for both Node.js and .NET Core. 
 + Changed to always generate bash and PowerShell scripts to enable cross-platform development.
-+ Added tasks.json to enable calling the scripts from VS Code.
```

Comment:
I've been talking with ScottHu about how they announce stuff on .NET, and what they've learned. 
If they want to see the code changes, they can look at the history, because it's open source. The version history should be why, and what experience changed.